react-debugging.html

```diff
@@ -62,10 +62,15 @@
           }
           // No need to change anything ABOVE this line ^
 
+// added an opening div on line 70
+// added an opening and closing tag on line 69 and 87. Stuck on how to solve it from here.
```

Comment:
There was actually no need for the div you added on line 70 since you added in the react fragments.  There are two more missing pieces, that your errors in the console should point you to.  Hint: an onclick should be calling a method from within the class.